.eslintrc.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
     plugins: ['@cloudflight/typescript'],
     extends: ['plugin:@cloudflight/typescript/recommended'],
     ignorePatterns: ['jest.config*.ts'],
+    rules: {
+        // This package is published, so private should not be forced
+        '@cloudflight/typescript/package-force-private': 0,
```

Comment:
don't think disabling this in the root project is correct, because it already has `private` set to true and it should not be published anyway.

Reply:
Enabled it again for all projects